src/content/docs/new-relic-solutions/best-practices-guides/full-stack-observability/synthetic-monitoring-best-practices-guide.mdx

```diff
@@ -267,6 +267,96 @@ console.log("Duration: " + script_duration)
 
     </Step>
 
+    <Step>
+    ## Configure and develop a scripted API test [#scripted-api-test]
+    Using scripted API monitors, you can build complex monitoring workflows with Node.js and the `got` module. For example, you can authenticate with an API, handle errors based on the response code, and parse the response body. To do this, you'll:
+
+    1. Go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Synthetic Monitoring**</DoNotTranslate>.
+    2. Click the <DoNotTranslate>**Create monitor**</DoNotTranslate> button.
+    3. Enter the name and details of your monitor. For example, `sitename.com`.
+    4. Select the locations from where you want your monitor to run. For example, Mumbai, Seoul, Columbus, or Montreal.
+    5. Choose a frequency for how often your monitor will run on each location selected. For example, `5 mins`.
+    6. Set a notification method to alert your team when performance incidents occur.
```

Comment:
This step doesn't reflect the actual experience, I don't think. I see it's from the original wording above - is it a holdover from an older Synthetics UI or Alerts flow?

Reply:
What should it be?